src/main.ts

```diff
@@ -6,9 +6,6 @@ import './registerServiceWorker'
 import router from './router/index'
 import store from './store'
 
-let socket = new WebSocket('ws://localhost:80')
```

Comment:
where has this gone?

Reply:
in the store, you have a socket module now